huaweicloud/services/acceptance/cc/resource_huaweicloud_cc_network_instance_test.go

```diff
@@ -203,18 +203,32 @@ func testNetworkInstance_multiple(name string, count int) string {
 	return fmt.Sprintf(`
 %[1]s
 
-resource "huaweicloud_cc_network_instance" "test" {
-  count = %[2]d
-
+resource "huaweicloud_cc_network_instance" "test1" {
   type                = "vpc"
   cloud_connection_id = huaweicloud_cc_connection.test.id
-  instance_id         = huaweicloud_vpc.test[count.index].id
-  project_id          = "%[3]s"
-  region_id           = huaweicloud_vpc.test[count.index].region
-
+  instance_id         = huaweicloud_vpc.test[0].id
+  project_id          = "%[2]s"
+  region_id           = huaweicloud_vpc.test[0].region
+  
   cidrs = [
-    huaweicloud_vpc_subnet.test[count.index].cidr,
+    huaweicloud_vpc_subnet.test[0].cidr,
   ]
 }
-`, testNetworkInstanceRef(name, count), count, acceptance.HW_PROJECT_ID)
+  
+resource "huaweicloud_cc_network_instance" "test2" {
+  type                = "vpc"
+  cloud_connection_id = huaweicloud_cc_connection.test.id
+  instance_id         = huaweicloud_vpc.test[1].id
+  project_id          = "%[2]s"
+  region_id           = huaweicloud_vpc.test[1].region
+  
+  cidrs = [
+    huaweicloud_vpc_subnet.test[1].cidr,
+  ]
+
+  depends_on = [
+    huaweicloud_cc_network_instance.test1,
+  ]
+}
+`, testNetworkInstanceRef(name, count), acceptance.HW_PROJECT_ID)
```

Comment:
Put depends_on in the front.
```suggestion
resource "huaweicloud_cc_network_instance" "test2" {
  depends_on = [huaweicloud_cc_network_instance.test1]
  
  type                = "vpc"
  cloud_connection_id = huaweicloud_cc_connection.test.id
  instance_id         = huaweicloud_vpc.test[1].id
  project_id          = "%[2]s"
  region_id           = huaweicloud_vpc.test[1].region
  
  cidrs = [
    huaweicloud_vpc_subnet.test[1].cidr,
  ]
}
`, testNetworkInstanceRef(name, count), acceptance.HW_PROJECT_ID)
```